maxim/logger/livekit/gemini/gemini_realtime_session.py

```
@@ -57,128 +75,142 @@ def handle_build_connect_config_result(
+    turn = session_info.current_turn
     if turn is None:
         return
     system_prompt = ""
-    if result.system_instruction is not None:
-        # We have to iterate through the system instructions
-        for part in result.system_instruction.parts:
-            system_prompt += part.text
+    if result.system_instruction is not None and isinstance(
+        result.system_instruction, Content
+    ):
+        if result.system_instruction.parts is not None:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)